repos/fdbt-site/src/pages/searchOperators.tsx

```diff
@@ -318,8 +372,8 @@ export const getServerSideProps = async (ctx: NextPageContextWithSession): Promi
             ];
         }
     }
-
-    return { props: { errors, searchText, searchResults, selectedOperators, csrfToken } };
+    const preSelectedOperators = selectedOperators;
```

Comment:
what is the purpose of this line - can you do this maybe?
return { props: { errors, searchText, searchResults, preSelectedOperators : selectedOperators , csrfToken } };